azure-pipelines.yaml

```diff
@@ -1,7 +1,9 @@
-name: $(Build.SourceBranchName)_$(Build.Reason)_$(majorVersion).$(Build.BuildId)$(Rev:.r)
+name: $(Build.SourceBranchName)_$(Build.Reason)_$(Major).$(Minor).$(Patch).$(Build.BuildId)$(Rev:.r)
 variables:
   buildConfiguration: 'Release'
-  majorVersion: 0.0.0
+  Major: '0'
+  Minor: '0'
+  Patch: '0-preview'
```

Comment:
according to our guidelines for pre-release packages published here - 
https://docs.microsoft.com/en-us/nuget/reference/package-versioning#version-basics

we should use one of the following conventions
- -alpha
- -beta
- -rc

I would say, in the current state, while under development, we should mark as -alpha. 
So 0.0.01-alpha is a good start. 

Once we're ready for people to use it and provide feedback we should mark it as -beta

Reply:
Do Patch version numbers auto increment on CI? Or only Minor on CD build? 
@jeffhollan & @anirudhgarg what is your convention for other extensions?

Reply:
What you said sounds reasonable. we can start with -alpha.  Waiting for the answer from them. I'll fix it.

Reply:
Ok

Reply:
Done.